runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -79,13 +79,13 @@ private void init()
 		add(food(15), PREMADE_TTL, TANGLED_TOADS_LEGS, PREMADE_CHOC_BOMB, CHOCOLATE_BOMB, COOKED_JUBBLY);
 		add(food(16), MONKFISH, POTATO_WITH_CHEESE, EGG_POTATO);
 		add(food(17), MYCIL_FISH_4, KRYKET_BAT_4);
-		add(food(18), COOKED_KARAMBWAN);
+		add(food(18), COOKED_KARAMBWAN, BLIGHTED_KARAMBWAN);
 		add(food(19), CURRY, UGTHANKI_KEBAB, UGTHANKI_KEBAB_1885);
 		add(food(20), MUSHROOM_POTATO, SHARK, ROQED_FISH_5, MURNG_BAT_5, STUFFED_SNAKE);
 		add(food(21), SEA_TURTLE);
-		add(food(22), MANTA_RAY, DARK_CRAB, TUNA_POTATO);
+		add(food(22), BLIGHTED_MANTA_RAY, MANTA_RAY, DARK_CRAB, TUNA_POTATO);
```

Comment:
I guess the blighted entry should be after the normal one, like in the other lines where a blighted version is added?
```suggestion
		add(food(22), MANTA_RAY, BLIGHTED_MANTA_RAY, DARK_CRAB, TUNA_POTATO);
```